### ISSUE TITLE
Buffer error with Axios on Server

### ISSUE THREAD

**Issue by devmondo** [fuse-box/fuse-box#103]
using Axios NPM package works on the client, but on the server,  this error pops up `TypeError: "list" argument must be an Array of Buffers`

and it is related to this part of the code in the `buffer` package 
```
 var buffer = Buffer.allocUnsafe(length)
  var pos = 0
  for (i = 0; i < list.length; ++i) {
    var buf = list[i]
    if (!(buf instanceof Buffer)) {
      throw new TypeError('"list" argument must be an Array of Buffers')
    }
```

**Comment by nchanged** [comment]
How are you initializing it? A Sample code? Does it happen on just `require`?

**Comment by devmondo** [comment]
sure
```
import axios from 'axios';

  async serve(url, clientId) {
      let result =await this.httpClient.get(`api/serve?url=${url}`, {
              headers: {'X-API-Client': clientId }
       });
       return result;
    }
```

the error happens when making the get call above.

**Comment by nchanged** [comment]
I need to have the entire project then. Or at least the part to run it.

**Comment by nchanged** [comment]
Fixed in `1.3.305`

**Comment by devmondo** [comment]
tested and works smoothly :)
